apisix/plugins/forward-auth.lua

```diff
@@ -117,7 +117,8 @@ function _M.access(conf, ctx)
 
     local httpc = http.new()
     httpc:set_timeout(conf.timeout)
-
+    local client_body_reader, err = httpc:get_client_body_reader()
+    params.body=client_body_reader
```

Comment:
Why do you add this?

Reply:
Agree with @monkeyDluffy6017 . Additionally, if this is a GET request, this approach may not be appropriate.

Reply:
@Gallardot `conf.timeout` or other conf parameters do not fix this issue and the root cause seems mysterious; I haven't been able to find it out.

> Additionally, if this is a GET request, this approach may not be appropriate.

Not a big deal, a condition check can be used. That said, there should be a better way to address this.

Reply:
Determine the request method or request body size, but this threshold is not easy to determine